MIPS/mip-0005-mip-upgrade.md

```diff
@@ -0,0 +1,315 @@
+
+**Review Phase - Part 1: Risk Assessment**
+We propose to create a pool of experts to review MIPs across the 3 categories. For each MIP, the Author reaches out to the pool of experts for the relevant category and sends a Request for Comments. To maintain independence, the Author does not choose who the Reviewers are. This MIP Review Group has a period to review the MIP and provide their feedback. The MIP Author can incorporate the feedback into the MIP until a final draft is completed. If the MIP Author does not incorporate the feedback, the MIP process still continues.
+
+When the MIP Author considers the draft ready for the final review, editing stops for at least 1 week to allow MIP Reviewers to write individually a Risk Assessment that is incorporated into the designated section of the MIP template. 
+
+The MIP Facilitator must check that every MIP has enough reviews to complete the Review phase (at least 3 if there is agreement, 5 otherwise). The MIP Facilitator can also coordinate communications and collaborate with the MIP Author to schedule any necessary meetings, and to define a roadmap with target dates for each milestone in the MIP process. 
+
+**Review Phase - Part 2: Community Conversation with Mass Deliberation**
+The MIP Facilitator sets a review period for the community to review the MIP, including the Risk Assessment, and schedules an open call where the MIP Author and MIP Review Group can present the proposal in detail. During this call, they will discuss the risks, benefits, and tradeoffs, address questions from the community, and provide instructions for on-chain voting, including the voting dates.
+
+After this meeting, the community will have some time again to reconsider the proposal before the voting takes place.
+
+https://drive.google.com/file/d/1RWpwc08Oj43zE25oKRdSmh-Gud_8RNfl/view?usp=drive_link
+
+### Guidelines for Implementation
+
+Every MIP should include an implementation plan that explicitly states what exactly needs to be done to implement the proposed changes, who is responsible for this implementation (in agreement with the Dev Team) and what are the target dates for each milestone. 
+
+MIP Authors are responsible for writing the implementation plan, collaborating and establishing connections with the Dev Team and other stakeholders during the implementation. 
+
+As a general rule, unless the MIP says otherwise, the implementation should be finished no later than one year after the approval via on-chain vote, and follow an iterative process with short delivery cycles (less than 3 months). 
+
+The Dev Team designated for the implementation of each MIP must provide transparency about how the implementation is being executed, so that the community can verify that the changes are implemented in the way and time that they were intended. 
+
+A comprehensive suite of tests is mandatory. When necessary, they will properly communicate the rationale for potential changes and delays.
```

Comment:
Why is this governance proposal overstepping into software development practices. While this is a good rule, it seems misplaced.

Reply:
I understand why it could seem misplaced, the intention here was to incorporate some lessons learned from previous experiences in the implementation of approved MIPs. Would you suggest leaving this for a more specific MIP treated separately?